eva-uow/src/main/kotlin/com/razz/eva/uow/ModelParam.kt

```diff
@@ -55,14 +55,7 @@ class ModelParam<MID : ModelId<out Comparable<*>>, M : Model<MID, *>> private co
             return modelParam
         }
 
-        fun <MID : ModelId<out Comparable<*>>, M : Model<MID, *>> modelParam(
```

Comment:
now it is impossible to wrap model in `modelParam` without `InstantiationContext`